features/step_definitions/authentication/sso_token_steps.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Given "the provider generated a sso token for the buyer" do
+  @sso_token = FactoryBot.build(:sso_token, account: @provider)
+  @sso_token.save
+end
+
+Given "another provider generated a sso token for another buyer" do
+  buyer = FactoryBot.create(:buyer_account)
+  @sso_token = FactoryBot.build(:sso_token, account: buyer.provider_account, user_id: buyer.users.first.id)
+  @sso_token.save
```

Comment:
Couldn't you just do `.create` and avoid `.save` ?

Reply:
No, because `Factorybot.create` calls `#save!` which is not implemented by `SSOToken` as it doesn't inherit from `ActiveRecord`.

Reply:
OK, thanks for the clarification :ok_hand:

Reply:
Then is it appropriate to use FactoryBot to begin with? I thought it is supposed to be only used for models. Or am I mistaken?

Reply:
There isn't a clear answer, `sso_token.rb` is under `/models` and implements some `ActiveRecord` features like validation, but it's not based on a DB table. So it's somewhere between a plain object and a Rails model. It seems to me that using FactoryBot or not is equally correct.